### ISSUE TITLE
WF does not overwrite

### ISSUE THREAD

**Issue by ThaDoctor72** [Pinacolada64/ImageBBS#23]
We need a better port of this.  The 8/3/89 version for 1.2 allowed you to overwrite if exist, and use alternate filenames at the time of saving. It would also be useful to give the user a choice to stay in editor or return to main.

**Comment by x-tec2017** [comment]
WF does overwrite.
When using the WF editor in 2.0, the .S (save) command is inoperative by design.
To save a file in the WF editor, you must use the .P (Put) command.

**Comment by ThaDoctor72** [comment]
We can mark this as closed. I ported over the working one from 1.2, and added the subroutine that checks for existing files.  Shall I attach it as a d64 image?

**Comment by x-tec2017** [comment]
No. The wf editor uses .P to put a file instead of .S to save. Functionality is the same but Put serves another purpose.  Leave it at is.


Sent from my T-Mobile 4G LTE Device
-------- Original message --------From: Hoy Brothers <notifications@github.com> Date: 9/3/18  8:44 PM  (GMT-05:00) To: Pinacolada64/ImageBBS <ImageBBS@noreply.github.com> Cc: x-tec2017 <larryhedman@gmail.com>, Comment <comment@noreply.github.com> Subject: Re: [Pinacolada64/ImageBBS] WF does not overwrite (#23) 
We can mark this as closed. I ported over the working one from 1.2, and added the subroutine that checks for existing files.  Shall I attach it as a d64 image?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Pinacolada64/ImageBBS","title":"Pinacolada64/ImageBBS","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/Pinacolada64/ImageBBS"}},"updates":{"snippets":[{"icon":"PERSON","message":"@ThaDoctor72 in #23: We can mark this as closed. I ported over the working one from 1.2, and added the subroutine that checks for existing files.  Shall I attach it as a d64 image?"}],"action":{"name":"View Issue","url":"https://github.com/Pinacolada64/ImageBBS/issues/23#issuecomment-418210741"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/Pinacolada64/ImageBBS/issues/23#issuecomment-418210741",
"url": "https://github.com/Pinacolada64/ImageBBS/issues/23#issuecomment-418210741",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "Re: [Pinacolada64/ImageBBS] WF does not overwrite (#23)",
"sections": [
{
"text": "",
"activityTitle": "**Hoy Brothers**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@ThaDoctor72",
"facts": [

]
}
],
"potentialAction": [
{
"name": "Add a comment",
"@type": "ActionCard",
"inputs": [
{
"isMultiLine": true,
"@type": "TextInput",
"id": "IssueComment",
"isRequired": false
}
],
"actions": [
{
"name": "Comment",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueComment\",\n\"repositoryFullName\": \"Pinacolada64/ImageBBS\",\n\"issueId\": 23,\n\"IssueComment\": \"{{IssueComment.value}}\"\n}"
}
]
},
{
"name": "Close issue",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueClose\",\n\"repositoryFullName\": \"Pinacolada64/ImageBBS\",\n\"issueId\": 23\n}"
},
{
"targets": [
{
"os": "default",
"uri": "https://github.com/Pinacolada64/ImageBBS/issues/23#issuecomment-418210741"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 375366523\n}"
}
],
"themeColor": "26292E"
}
]

**Comment by x-tec2017** [comment]
This is a non-issue. It can be closed.
